gobblin-salesforce/src/main/java/org/apache/gobblin/salesforce/SalesforceExtractor.java

```diff
@@ -114,7 +115,7 @@
   private boolean pullStatus = true;
   private String nextUrl;
 
-  private BulkConnection bulkConnection = null;
+  protected BulkConnection bulkConnection = null;
```

Comment:
Do you still need to relax this to protected?

Reply:
fixed: change back to private